Kudu.Core/Permissions/PermissionHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.IO;
+using Kudu.Contracts.Permissions;
+using Kudu.Contracts.Settings;
+using Kudu.Core.Deployment;
+using Kudu.Core.Deployment.Generator;
+using Kudu.Core.Infrastructure;
+
+namespace Kudu.Core.Permissions
+{
+    /// <summary>
+    /// This handler is meant for Linux enviroment only
```

Comment:
static function was my first implementation, but since i will need Enviroment, DeploymentSettingsManager and Logger, thought to follow the pattern to use Ninject. 

Let me refactor it to be static function.